src/main/java/com/redhat/documentation/asciidoc/extraction/SourceExtractor.java

```diff
@@ -320,7 +323,7 @@ else if ("source".equals(blockAttributes.get("1")) && blockAttributes.containsKe
                 keys.stream()
                         // We don't really need the positional ones as they'll be listed as named (I think)
                         .filter(s -> s.matches("\\D+.*"))
-                        .forEach(key -> joiner.add(key + "=\"" + node.getAttribute(key).toString() + "\""));
+                        .forEach(key -> joiner.add(key + "=\"" + node.getAttribute(key) + "\"")); // TODO: getAttribute(key) can return null!
```

Comment:
this returns null for some atributes in quarkus docs...not sure if error in docs or error in assumption in this code.

Reply:
Yeah, I ran across this the other day with how attributes are processed.... I'm not happy about it, but it's a known deficiency in asciidoctor. Do you happen to know which doc this was happening with?

Reply:
Just ran it on Quarkus doc folder :)

Reply:
This should be fixed now as well.